projects/plugins/jetpack/modules/likes/style.css

```diff
@@ -217,3 +217,10 @@ div.sharedaddy.sd-like-enabled .sd-like .post-likes-widget {
 .sd-gplus .sd-title {
 	display: none;
 }
+
+/* Hide the share this section in the post print view */
+@media print {
+	div.sharedaddy.sd-sharing-enabled {
+	   visibility: hidden;
+	}
+ }
```

Comment:
Couple thoughts:

1) Is there a specific reason you used `visibility: hidden` instead of `display: none`? `visibility: hidden` leaves an empty space where the sharing buttons used to be, which is probably not what you want.
2) Spacing is a bit off—looks like the actual CSS rule uses spaces instead of tabs.
3) Nitpicky, but you probably don't need the `div`. Just `.sharedaddy.sd-sharing-enabled` should be enough.

Reply:
Thanks for your comments, @danielpost. I have updated the screenshots as well, I can see that the extra space is removed. 

1.  I have changed `visibility: hidden` to `display: none`. I didn't know that `visibility:hidden` leaves an empty space, my CSS isn't the best, will learn. :) 
2. Fixed the spacing.
3. Removed the div. prefix, I assumed that it was an A8C pattern to do it with the prefix, but doesn't appear to be.